tests/integration/features/test_reports/robot_xml/01_basic/strictdoc.toml

```diff
@@ -0,0 +1,11 @@
+[project]
+
+include_source_paths = [
+  "tests/**.robot",
+]
+
+features = [
+  "REQUIREMENT_TO_SOURCE_TRACEABILITY",
+  "SOURCE_FILE_LANGUAGE_PARSERS",
+  "PROJECT_STATISTICS_SCREEN"
```

Comment:
I guess we don't need the project statistics for this test? 

(some tests are messy in this regard but it is still less stuff to generate since I noticed it)